cfr.yaml

```diff
@@ -197,3 +197,65 @@
       - sling.com
       - www.facebook.com
       - facebook.com
+- id: better-internet-c-rollout:rollout-90
+  groups:
+    - spotlight
+  content:
+    template: logo-and-content
+    logoImageURL: https://www.mozilla.org/media/img/firefox/onboarding/mountain.bd39012cc5a9.svg
+    logo:
+      size: 115px
+    body:
+      title:
+        label: A better Internet starts with you
+        size: 24px
+      text:
+        label: When you use Firefox, you’re voting for an open and accessible Internet that’s better for everyone. So pin us to your taskbar to make sure we’re always front and center.
+        size: 16px
+      primary:
+        label: Pin to taskbar
+        action:
+          type: PIN_FIREFOX_TO_TASKBAR
+      secondary:
+        label: Not now
+        action:
+          type: CANCEL
+  trigger:
+    id: defaultBrowserCheck
+  template: spotlight
+  frequency:
+    lifetime: 1
+  # 1. Nimbus rollout enrollment targeting except sticky and study opt-out
+  # 2. Nimbus rollout message targeting
+  # 3. Additional message non-overlap
+  targeting: >-
+    userMonthlyActivity|length >= 1 &&
+    userMonthlyActivity|length <= 6 &&
+    (currentDate|date - profileAgeCreated|date) / 86400000 >= 28 &&
+    doesAppNeedPin  &&
+    browserSettings.update.channel == 'release' &&
+    version|versionCompare('94.!') >= 0 &&
+    locale in ['en-CA', 'en-GB', 'en-US']
+    &&
+    source == 'startup' &&
+    'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features'|preferenceValue &&
+    'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons'|preferenceValue &&
```

Comment:
I would move the prefs in the message group definition

Reply:
Looks like the message group prefs behavior is `some` and not `every` but maybe that's what we want anyway instead of strictly requiring both?

https://searchfox.org/mozilla-central/rev/9bed2623831804ac086bbb80cb666e5c3b00416a/browser/components/newtab/lib/ASRouter.jsm#709-712

Reply:
Sorry yes I was thinking about the "recommend features as I browser" pref only.